src/tests/flyer.test.ts

```diff
@@ -294,4 +293,41 @@ describe('deleteFlyer tests', () => {
     const deleteFlyerResponse = await FlyerRepo.deleteFlyer('64811792f910705ca1a981f8');
     expect(deleteFlyerResponse).toBeNull();
   });
-});
\ No newline at end of file
+});
+
+describe('getAllFlyerCategories tests', () => {
+  test('Ensure only unique CategorySlugs across flyers in collection are returned', async () => {
+    const flyerSlugs = ['Academic', 'Dance'];
+    const generatedFlyers = [];
+    generatedFlyers.push(
+      (await FlyerFactory.createSpecific(1, { categorySlug: flyerSlugs[0] }))[0],
+    );
+    generatedFlyers.push(
+      (await FlyerFactory.createSpecific(2, { categorySlug: flyerSlugs[1] }))[0],
+    );
```

Comment:
You may want to make this a for loop as well so the tests follow the same style
```TypeScript
for (const slug of flyerSlugs) {
      generatedFlyers.push((await FlyerFactory.createSpecific(1, { categorySlug: slug }))[0]);
    }
```

Reply:
I actually wouldn't recommend using a for loop here because the goal is to create 3 flyers, two of which have the same slug. The code above would only test 2 flyers with 2 unique slugs which is semantically what the other test does. You could technically make this a loop by making the categorySlugs array ["Category1", "Category2", "Category2"] but honestly I think the current code is a lot more readable

Reply:
Interesting, I would argue that the `["Category1", "Category2", "Category2"]` is more readable, since it clearly outlines which category slugs we are adding to `generatedFlyers` at the top of the function, as opposed to the current code, where one might skip over the `2` in the parameters of the `createSpecific` function (this is what I did while reading it). But we're just nitpicking at this point, it doesn't really matter which approach you use. That's why I said you _may_ want to make this a for loop. But imo it should be up to Cindy at this point.

Reply:
Wait just noticed that the current code actually only adds 2 distinct Flyers!
`(await FlyerFactory.createSpecific(2, { categorySlug: flyerSlugs[1] }))[0]`
only adds the Flyer at index 0 of the generated array of 2 Flyers. So she needs to either use the for-loop approach or use `generatedFlyers.concat` and remove the accessor.

Reply:
Oh yeah I had assumed this code was using `concat()` like my suggestion above haha.